test/java/seedu/addressbook/util/TestPerson.java

```diff
@@ -0,0 +1,59 @@
+package seedu.addressbook.util;
+
+import seedu.addressbook.data.person.Address;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Phone;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+public class TestPerson implements ReadOnlyPerson {
```

Comment:
Why do we need this class? In any case, there should be a header comment to explain what it is.